examples/cpp/function-execution/main.cpp

```diff
@@ -38,147 +34,114 @@
 #include <geode/FunctionService.hpp>
 #include <geode/CacheableString.hpp>
 
-// Use the "geode" namespace.
-using namespace apache::geode::client;
+using apache::geode::client::Cache;
+using apache::geode::client::CacheableArrayList;
+using apache::geode::client::CacheableKey;
+using apache::geode::client::CacheableString;
+using apache::geode::client::CacheableVector;
+using apache::geode::client::CacheFactory;
+using apache::geode::client::Exception;
+using apache::geode::client::FunctionService;
+using apache::geode::client::Region;
+using apache::geode::client::RegionShortcut;
+
+const auto getFuncIName = std::string("ExampleMultiGetFunction");
+
+const int EXAMPLE_ITEM_COUNT  = 6;
+const int EXAMPLE_SERVER_PORT = 50505;
+
+std::vector<std::string> keys = {
+    "KEY--1",
+    "KEY--2",
+    "KEY--3",
+};
+
+std::vector<std::string> values = {
+    "VALUE--1",
```

Comment:
Why not have the key and values be more relatable than this. Like how does "KEY--1" relate to "VALUE--3". Seems so random.

Reply:
Yeah, just changed values to 1,2,3 as well - makes more sense, at least they agree now.